### PR TITLE
add search term to query parameters

### PR DESCRIPTION
**Pull request by crhistianramirez**
### What
Adds search term to query parameters

### Why
Lets us:
* Track usage in google analytics
* Enables users to save searches via bookmarks
* Enables users to share searches as links

This completes part of https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/issues/158 I didn't quite have enough time to add filters but I hope to work on that tomorrow. At least this way we can start tracking search

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [x] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari

### CODE REVIEW

**Comment by benjifs** [comment]
Not sure if this is a legitimate issue but I'll just point it out because I noticed it in my testing of this feature.

This takes whatever you put into the query parameter and puts it in the input box. While that's ok generally, we might want to encode it on the way in or something. 

Try searching for the following string:
`"><h1 onclick="alert(1)">hello</h1>`

If you add it to the search box you get no results. If you add it to the search so that the url ends in:
`?search="><h1 onclick="alert(1)">hello</h1>`

You will now see "**hello**" on screen that when clicked alerts "1".

For our use case I don't know how this could be exploited but I'd be more comfortable addressing this before it goes out.

**Comment by experimatt** [comment]
At first glance this looks good. I tested it out in the Netlify preview and it worked as expected.

The implementation of `js/url-helpers.js` at first glance feels longer/more complicated than I would have expected but that's probably bikeshedding and doesn't actually matter. Nice work, @crhistianramirez!

I've gotta log off for the night so I'll let someone else give the official approval & merge

**Comment by crhistianramirez** [comment]
@benjifs Very valid point - cross site scripting is no joke. I'm now sanitizing user inputs.

File: src/js/url-helpers.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export function getQueryParam(paramName) {
```
Comment (jdalt) [inline src/js/url-helpers.js:1]: Nice to see a small focussed, file. We need to break more of `script.js` into something like this.

**Pull request merged**